src/@next/Tag/Tag.tsx

```diff
@@ -17,5 +30,21 @@ export const Tag = ({ children, ...props }: TagProps) => {
       children
     );
 
-  return <TagStyle {...props}>{content}</TagStyle>;
+  const removeButton = onRemove ? (
```

Comment:
```suggestion
  const removeButton = onRemove && (
```

we can do this instead of the ternary operator since onRemove is a function

Reply:
resoved